Common.mk

```diff
@@ -163,7 +163,8 @@ HELM_SOURCE_REPOSITORY?=$(REPO)
 HELM_SOURCE_IMAGE_REPO?=$(IMAGE_REPO)
 HELM_GIT_TAG?=$(GIT_TAG)
 HELM_TAG?=$(GIT_TAG)-$(GIT_HASH)
-HELM_DIRECTORY?=.
+HELM_USE_UPSTREAM_IMAGE?=false
```

Comment:
See https://github.com/aws/eks-anywhere-build-tooling/pull/1287/files#r974240967

Reply:
yeah, you're right. Here's the build changes: https://github.com/aws/eks-anywhere-build-tooling/pull/1315